src/python/NOTES.md

```diff
@@ -2,6 +2,6 @@
 
```

Comment:
Can you help fix the failing test https://github.com/devcontainers/features/actions/runs/10442944409/job/28957430957?pr=1015#step:4:70491 ?

Thanks!

Reply:
Should be fixed. Had an issue with the `grep` install/upgrade that's now resolved.